src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -442,8 +442,13 @@ private function registerHttpCache(ContainerBuilder $container, array $config, X
 
         $definitions = [];
         foreach ($config['http_cache']['invalidation']['varnish_urls'] as $key => $url) {
+            $clientConfig = [
+                'base_uri' => $url,
+                'verify' => $config['http_cache']['invalidation']['verify_ssl'],
+            ];
+
             $definition = new ChildDefinition('api_platform.http_cache.purger.varnish_client');
-            $definition->addArgument(['base_uri' => $url]);
+            $definition->addArgument($clientConfig);
```

Comment:
You can inline this directly, it does not need to be a variable.

Reply:
Indeed, I modify my PR.

Thanks.